docker-build.sh

```diff
@@ -0,0 +1,14 @@
+#!/usb/bin/sh
+
```

Comment:
Image build infrastructure.
This will pull tags from git tags and push images to docker hub